README.md

```diff
@@ -37,6 +37,34 @@ Se ainda estás a dar os primeiros passos no Github, eis alguma documentação 
 - [Como fazer fork de um repositório](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo)
 - [Como fazer um pull request](https://docs.github.com/pt/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
 
+### Como adicionar novas localizações ao mapa?
+A informação apresentada no mapa é recolhida a partir de entradas existentes no [Wikidata](www.wikidata.org).
+
+#### Para adicionar uma nova entrada
+1. No menu lateral, escolher "Criar um objecto novo" e acrecentar a informação nos campos obrigatórios *NOTA: A língua tem de ser pt ou en*
+2. Depois de criar o objecto, temos de adicionar propriedades que correspondam aos filtros usados para popular o mapa. Escolher **adicionar declaração** e depois procurar as propriedades relevantes. Listamos as mandatórias:
+   1. **país** - Portugal
+   2. **coordenadas geográficas** - introduzir as coordenadas do local
+   3. **instância de** - define a categoria do local. Escolher para as categorias do mapa:
+      1. **Bibliotecas** - biblioteca OU biblioteca municipal
+      2. **Cinemas** - sala de cinema
+      3. **Galerias** - galeria de arte
+      4. **Monumentos** - monumento
+      5. **Museus** - museu
+      6. **Recintos** - casa de shows
+      7. **Teatros** - teatro
+   4. Quando escolher o valor para cada propriedade, carregar em *publicar*
+   5. Podem enriquecer os dados do local adicionado com outras categorias que achem relevantes
+
+*NOTA: É provável que algumas localizações já existam no Wikidata, mas apenas não estejam associadas às categorias acima indicadas.*
```

Comment:
Eu aqui acrescentaria a recomendação de pesquisar antes de criar o novo objeto (o que justificaria mover esta linha para o início desta secção), e talvez mencionar que não há problema se duplicados forem criados, porque podem ser fundidos posteriormente.

Reply:
Concordo. Mudei o texto para o primeiro item da lista e transformei a nota no aviso sobre os duplicados, @waldyrious